ra-optimisation/linear-range-action-rao/src/test/java/com/farao_community/farao/linear_range_action_rao/JsonLinearRangeActionRaoResultTest.java

```diff
@@ -34,4 +40,25 @@ public void roundTripTest() {
         //compare
         compareTxt(getClass().getResourceAsStream("/LinearRangeActionRaoResults.json"), baos.toString());
     }
+
+    @Test
+    public void anotherRoundTest() {
+        RaoComputationResult outputResult = new RaoComputationResult(RaoComputationResult.Status.SUCCESS, null, null);
+        LinearRangeActionRaoResult outputResultExtension = new LinearRangeActionRaoResult(LinearRangeActionRaoResult.SecurityStatus.UNSECURED);
+        double minMargin = -1;
+        outputResultExtension.setMinMargin(minMargin);
+        outputResult.addExtension(LinearRangeActionRaoResult.class, outputResultExtension);
+
+        OutputStream outputStream = new ByteArrayOutputStream();
+        JsonRaoComputationResult.write(outputResult, outputStream);
+        assertTrue(outputStream.toString().contains("minimum-margin"));
+
+        InputStream inputStream = new ByteArrayInputStream(outputStream.toString().getBytes());
+        RaoComputationResult inputResult = JsonRaoComputationResult.read(inputStream);
+        LinearRangeActionRaoResult inputExtension = inputResult.getExtension(LinearRangeActionRaoResult.class);
+        assertEquals(RaoComputationResult.Status.SUCCESS, inputResult.getStatus());
+        assertEquals(LinearRangeActionRaoResult.SecurityStatus.UNSECURED, inputExtension.getSecurityStatus());
+        assertTrue(inputExtension.getMinMargin() == minMargin);
```

Comment:
Can be replaced by assertEquals as it is double values

Reply:
ok